### ISSUE TITLE
Retrieve Standard Objects in Org Browser

### ISSUE THREAD

**Issue by banderson5144** [forcedotcom/salesforcedx-vscode#1542]
### Summary

Retrieve standard objects for Org Browser

### Steps To Reproduce:

1.  Refresh list of CustomObject metadata type

### Expected result

Should see list of Standard and Custom objects

### Actual result

Just list of Custom Objects

### Additional information

**VS Code Version**: 1.36.1

**SFDX CLI Version**: 7.15.3

**OS and version**: Windows 10

**Comment by suraj07081995** [comment]
Run this code to see the list of Standard Objects and Custom Objects
`Map<string,Schema.sObjectType> standardObj = Schema.getGlobalDescribe();`
`for(string str : standardObj.keyset() ){`
`if(!str.contains('__')){`
     `System.debug('Standard Object ==>>'+str);`
  `}`
`}`
`for(string str : standardObj.keyset() ){`
`if(str.contains('__')){`
  `System.debug('Custom Object ==>>'+str);`
`}`
`}`
I hope this helps you!

**Comment by jvaloto** [comment]
If you want to retrieve this components in metadata, you have to add this to your package.xml (account e.g):

```
<types>
      <members>Account</members>
      <name>CustomObject</name>
   </types>
```

**Comment by brpowell** [comment]
A fix for this was merged in #1571 and addressed in today's release 46.11.0
